NEWS.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.2.7] - 2022-10-07
+
+### Added
+- Added `DistributedCellDof`, a distributed wrapper for `Gridap.CellDof`. This new wrapper acts on `DistributedCellField` in the same way `Gridap.CellDof` acts on `CellField`. Added `get_fe_dof_basis` function, which extracts a `DistributedCellDof` from a `DistributedFESpace`. Since PR [#97](https://github.com/gridap/GridapDistributed.jl/pull/97).
+- Added `gather_free_and_dirichlet_values!` and `gather_free_values!` wrapper functions. Since PR [#97](https://github.com/gridap/GridapDistributed.jl/pull/97).
```

Comment:
Something I am wondering now is the following. In `Gridap.jl`, `scatter_free_and_dirichlet_values` is the inverse of ``gather_free_and_dirichlet_values!`. Why we dont need `scatter_free_and_dirichlet_values` in `GridapDistributed.jl`?

As far as I can see it has to do with the fact that in `GridapDistributed.jl` we dont reuse the `FEFunction` constructor from `Gridap.jl`, but re-implement it using different tools.

Reply:
Yeah, the implementation of the distributed counterpart of `FEFunction` is quite weird. And indeed the constructor does not scatter the free values. What would be the reason for that?

Reply:
> What would be the reason for that?

The distributed variant of the `FEFunction` constructor ends in the following private function:

```julia 
function _EvaluationFunction(func,
  f::DistributedSingleFieldFESpace,free_values::AbstractVector,
  dirichlet_values::AbstractPData{<:AbstractVector},isconsistent=false)
  local_vals = consistent_local_views(free_values,f.gids,isconsistent)
  fields = map_parts(func,f.spaces,local_vals,dirichlet_values)
  metadata = DistributedFEFunctionData(free_values)
  DistributedCellField(fields,metadata)
end
```

Scatter actually occurs locally at each processor in the following line `fields = map_parts(func,f.spaces,local_vals,dirichlet_values)`. 

There does not seem to be the need to generate an `AbstractPData` instance that holds inside the results of scattering locally on each processor, i.e., for a distributed version of `scatter_free_and_dirichlet_values`. 

On another note, note also the `isconsistent` keyword parameter in the distributed `FEFunction` constructor. Whenever it is true, it lets you guarantee that the ghosts dofs are consistent among processors, i.e., a communication is performed in order to gather the ghost dof values from neighbouring processors.